ui/server/src/test/scala/pl/touk/nussknacker/ui/process/migrate/ProcessModelMigratorSpec.scala

```diff
@@ -34,6 +34,16 @@ class ProcessModelMigratorSpec extends FlatSpec with BeforeAndAfterEach with Pat
     processor shouldBe ServiceRef(ProcessTestData.otherExistingServiceId, List())
   }
 
+  it should "migration should not return empty migration result" in {
+
+    val migrationResultOpt: Option[MigrationResult] = migrateByVersionsOpt(Some(1), 1)
+
+    migrationResultOpt.map(o => println(o.toUpdateAction(ProcessId(1L))))
+
+    migrationResultOpt shouldNot be('defined)
```

Comment:
Easier would be `shouldBe empty`

Reply:
Changed.
Personally I like using this format for ex. `be('defined)` since It implies that I'm dealing with optionals at the first glance.

Reply:
you've right but you have non necessary negation, but maybe it is subjective thing